module/VuFindSearch/src/VuFindSearch/Response/SimpleRecord.php

```diff
@@ -50,6 +47,13 @@ class SimpleRecord implements RecordInterface
      */
     protected $fields = [];
 
+    /**
+     * Labels for the record
+     *
+     * @var array
+     */
+    protected $labels = [];
```

Comment:
I think this is redundant (also in RecordTrait).

Reply:
Fixed.